recipes/openff-toolkit/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "openff-toolkit" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openforcefield/openforcefield/archive/{{ version }}.tar.gz
+  sha256: cd4ca0ded60ebd97120523b85c2aa6e0ddbc400193e6bbf49aa52b82ccf9e026
```

```suggestion
  sha256: 20597c0e325ad76d1bf161eb88717c397af5d1d794f4b694d28784b8e38d7560
```

Assuming I remembered the right way to get the sum:
```
wget https://github.com/openforcefield/openforcefield/archive/0.8.1.tar.gz
sha256sum 0.8.1.tar.gz
```